master/internal/grpcutil/auth.go

```diff
@@ -33,6 +33,10 @@ var unauthenticatedMethods = map[string]bool{
 	"/determined.api.v1.Determined/GetTelemetry": true,
 }
 
+var adminMethods = map[string]bool{
+	"/determined.api.v1.Determined/DeleteExperiment": true,
```

Comment:
non-blocking: Would it be feasible to do something like use the API annotations for this sort of thing (along the lines of the current `tags: "Experiments"`) rather than this separate thing? This'll fail insecure if anything changes or we add new APIs that should be protected. Probably not a big deal yet, but I twitch a bit at seeing this.

Reply:
I mean, this is just an access control list lol, just a crummy one. Yeah, having at the API layer would be neat, but I was moving with the assumption something like that would get wrapped into an RBAC project.